### DISCUSSION TITLE
Export pcaps

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#8316] [Unsupported Versions] [answered]
Hi,
Does anyone know how to collect pcaps in the most efficient way on a regular basis?
Say I want to take a month worth of pcaps, how would I go about it?
I know that I need to use so-pcap-export but how would I take a month back using the Steno query language?

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/stenographer.html#command-line:
> You can also access packet capture from the command line of the box where the pcap is stored using a steno query as defined at https://github.com/google/stenographer#querying. 

If you follow the link to https://github.com/google/stenographer#querying, you should see some examples of setting your desired time frame:
![image](https://user-images.githubusercontent.com/1659467/179760211-7c3f6022-2d72-4a57-b132-ac7a6457ab5a.png)
